sys-utils/unshare.c

```diff
@@ -213,6 +214,23 @@ static ino_t get_mnt_ino(pid_t pid)
 	return st.st_ino;
 }
 
+static void settime(time_t offset, clockid_t clk_id)
+{
+	char buf[4096];
```

Comment:
What about 
     char buf[ sizeof(stringify_value(ULONG_MAX)) * 3  ];
rathe rthan the magic number?

Reply:
done